website_typed_text/templates/snippets.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <!-- Snippet options -->
+    <template id="snippet_options" inherit_id="website.snippet_options">
+        <xpath expr="//div" position="after">
+            <div data-js="typed_text" data-selector="*:not(:has(*))">
+                <we-button data-toggle-class="typed_text" data-no-preview="true">
```

Comment:
What is 'we-button'? first time that i see this.

Reply:
In 13 this the option element I don't know the semantics of it